homeassistant/components/weatherflow/sensor.py

```diff
@@ -265,17 +265,17 @@ def get_native_value(self, device: WeatherFlowDevice) -> datetime | StateType:
         device_class=SensorDeviceClass.WIND_SPEED,
         icon="mdi:weather-windy",
         event_subscriptions=[EVENT_RAPID_WIND, EVENT_OBSERVATION],
-        native_unit_of_measurement=UnitOfSpeed.KILOMETERS_PER_HOUR,
+        native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
         state_class=SensorStateClass.MEASUREMENT,
         suggested_display_precision=2,
         raw_data_conv_fn=lambda raw_data: raw_data.magnitude,
     ),
     WeatherFlowSensorEntityDescription(
-        key="wind_speed_average",
+        key="wind_average",
```

Comment:
These are part of the unique ID of entities. This requires an migration of entities in the registry that carry to old unique ID to the new one.

Reply:
Thanks, for clarifying 👍